src/components/BookList.js

```diff
@@ -0,0 +1,36 @@
+import PropTypes from 'prop-types';
+import Book from './Book';
+import './BookList.css';
+
+const BookList = ({ books }) => (
+  <div className="main-book-list">
+    <table>
+      <thead>
+
+        <tr>
+          <th>ID</th>
+          <th>TITLE</th>
+          <th>AUTHOR</th>
+          <th>CATEGORY</th>
+          <th>YEAR</th>
+          <th>DELETE</th>
+        </tr>
+      </thead>
+      <tbody>
+
+        {books.map((book, index) => <Book book={book} key={index.toString()} />)}
+      </tbody>
+    </table>
+  </div>
+);
+
+BookList.propTypes = {
+  // eslint-disable-next-line react/forbid-prop-types
```

Comment:
- [ ] We cannot use `eslint-disable-lines`. Instead, we need to conform with `eslint` given rules. A possible solution can be something like this:
```JS
books: PropTypes.arrayOf(PropTypes.shape({
    id: PropTypes.number.isRequired,
    title: PropTypes.string.isRequired,
    category: PropTypes.string.isRequired,
  })).isRequired,
```

Reply:
thank you so much for this tip.